apps/flame-defi/app/utils/utils.ts

```diff
@@ -63,33 +61,3 @@ export function formatDecimalValues(value?: string, decimals?: number): string {
   const decimalVal = decimals === undefined ? 4 : decimals;
   return value ? parseFloat(value).toFixed(decimalVal) : "0";
 }
-
```

Comment:
I can't comment on it because it's not part of changed code, but above you have "115792089237316195423570985008687907853269984665640564039457" hardcoded. This needs to come from our config `tokenDefaultApprovalAmount`

Reply:
Is that actually the right value to use for slippage tolerance?

Reply:
We can't pull in the tokenDefaultApprovalAmount into the utils code because the context comes from a hook which can't be used here since this file is not a hook.

Reply:
Good catch here